ui_framework/src/components/icon/icon.js

```diff
@@ -87,3 +88,7 @@ KuiIcon.propTypes = {
   size: PropTypes.oneOf(SIZES),
   title: PropTypes.string,
 };
+
+KuiIcon.defaultProps = {
+  className: 'kuiIcon--basic',
```

Comment:
I'm ok with this in the React sense and it totally makes sense. I'm a little worried in the HTML/CSS sense where suddenly default settings of css components are "broken". For example, what's the version of this for `.kuiButton`? Does that component not usable on its own? Seems so easy to just solve with `.kuiIcon.modifier`.

Just something to think about.